css/styles.css

```
@@ -318,26 +376,77 @@ p {
 
 .phase-items {    
     list-style: none;
-    font-family: Poppins, Helvetica, sans-serif;
     font-size: 18px;
-    font-weight: 400;
     line-height: 28px;
     letter-spacing: 0;
-    text-align: left;
     color: #c2c3c5;
+    text-transform: capitalizeupp;
+}
+
+.phase-items>p::before {
+    content: url(/img/logos/icon_w24h24.png);
+    margin-right: 8px;
+}
+
+.r-map-btns-arrows {
+    border: none;
+    background-color: transparent;
+}
+
+.arrow-left-roadmap {
+    transform: rotate(90deg);
 }
 
-.r-map-list-btns {
+.arrow-right-roadmap {
+    transform: rotate(-90deg);
+}
+
+.r-map-btns-arrows:hover {
+    fill: #21E786;
+}
+/* How else can I recolor the color of the svg arrow when hovering (because this method doesn't work properly) or should I just apply different images? I would like to use only styling. */
+
```

You chosen good way. Styling `fill` attribute is the best approach for SVG icons because only this way allows you to change the color smoothly via for example `transition: fill .2s ease-out`.

This method doesn't work here because you applied this class (`r-map-btns-arrows`) to `<button>` tag in markup. If you move specify SVG, it should help (or you can specify classname of icons instead of tag)

```suggestion
.r-map-btns-arrows:hover svg {
    fill: #21E786;
}
```